WooCommerce/Classes/ViewRelated/Dashboard/Settings/Help/SupportForm/SupportForm.swift

```diff
@@ -105,49 +106,110 @@ struct SupportForm: View {
     @StateObject var viewModel: SupportFormViewModel
 
     var body: some View {
-        VStack(spacing: Layout.sectionSpacing) {
-
-            HStack(spacing: -Layout.optionsSpacing) {
-                Text(Localization.iNeedHelp)
-                    .bold()
-                Picker(Localization.iNeedHelp, selection: $viewModel.area) {
-                    ForEach(viewModel.areas, id: \.self) { area in
-                        Text(area.title).tag(area)
+        ZStack() {
+            // Background
+            Color(.listBackground)
+                .frame(maxWidth: .infinity, maxHeight: .infinity)
```

Comment:
Kinda 😅 , I wanted the background to extend all the way (even beyond safe areas) and not be dependent on the internal stack size, which is dynamic and variable.  

But in practice, since the content will always be greater than the screen size I think both approaches would look the same.

Reply:
In any case, adjusted it on https://github.com/woocommerce/woocommerce-ios/pull/8869/commits/4e18085d8a7bca16e9c3b9734c74c0af79644c27 as it looks simpler! :-)

Reply:
Agreed, this looks simpler, but thanks for explaining the reasoning — it's helpful to keep in mind for other situations!